ReadMe.md

```diff
@@ -87,6 +87,7 @@ To copy and paste | コピペ（する） | |
 English | Japanese | Kana | Notes
 --------|----------|------|------
 Device, terminal | 端末 | たんまつ | Smartphones, tablets, e-readers, command-line interfaces, etc
+Cell phone | ケータイ / 携帯電話　| けいたいでんわ | standard (non-smartphone) cell phone　
```

Comment:
I'll look into the spelling of keitai though, since there are 2 katakana styles and possibly even kanji. I would like the most common one represented.